### DISCUSSION TITLE
Getting 403's on Connection Creation.

### DISCUSSION THREAD

**Discussion by HauptmannEck** [planetscale/discussion#129] [Bug Reports] [answered]
After running perfectly for 4 months, today my app started failing to get data from Planetscale due to the `/create-certificate` endpoint returning 403 even though the settings in my Org seem to be the same.

I tried both v0.1.0 and v0.2.0 node package as I noticed the cert endpoint changed, but both returned the same 403. My tokens and service account is maintained by my Vercel integration, so I have not manually changed anything. I have 2 branches that I use, local vs prod environments, and neither are working.

Wondering if there is a solution that is not rebuilding my connection to planetscale.

Also since the 403 returns HTML instead of JSON the error handling breaks and cannot be handled properly as it takes down the entire NextJs app.
This does not seem to fire:
`req.on('error', (e) => reject(e));`

**Comment by jamesmosier** [comment, marked as answer]
Issue opened here: https://github.com/planetscale/planetscale-node/issues/31

**Comment by jgreet** [comment]
The certificate endpoint issue should be resolved in `v0.3.0` with https://github.com/planetscale/planetscale-node/pull/33.
